swagger.yaml

```diff
@@ -278,6 +278,249 @@ paths:
               schema:
                 $ref: "#/components/schemas/InternalServerError"
 
+  "/v1/templates":
+    get:
+      operationId: listTemplates
+      summary: Lists Templates
+      description: |
+        Lists Templates.
+        
+        Method allows public templates for all users but private templates require that user is logged in
```

Comment:
Permission description is not correct, maybe this is an old version of the spec?